src/lib/spinel/multi_frame_buffer.hpp

```diff
@@ -39,10 +39,14 @@
 #include <string.h>
 
 #include <openthread/error.h>
+#include "lib/utils/endianness.hpp"
```

Comment:
I tend to add a space before differenct sections of headers.
```suggestion

#include "lib/utils/endianness.hpp"
```

Reply:
Accept. Thanks!